### PR TITLE
🏇 Personal Profile: Update Profile Details

### LINKED ISSUE #138: 🏇 Personal Profile: Add the option to update
Right now, the personal profile only displays personal details from a detail view.
It is needed to add the option to edit those details if a user is looking at his own profile.

- [x] Create a simple UpdateView with a form
- [ ] Adding relevant tests
- [ ] Adding a button that redirects to the update form

### PR DESCRIPTION
**Pull request by ShirleyFichman**
In this PR I added a simple UpdateView with a relevant form and added the relevant tests.
Also added a button that redirects to the form.

Close #138

image

image

### CODE REVIEW

**Review by DeanBiton — APPROVED** [review]
LGTM

**Review by paOmer — APPROVED** [review]

**Review by amittcohen — APPROVED** [review]
LGTM

File: personal_profile/tests.py [hunk 1]
```diff
@@ -104,3 +105,29 @@ def test_detail_view_template(self, profile_1, user_1, client):
...
+        client.force_login(test_user)
+        update_profile_path = PROFILE_DETAIL_URL + str(test_profile.id) + UPDATE_PROFILE_PATH
+        new_fields = {
+            'company': 'Update Test Company',
+            'birth_date': datetime.date(1992, 12, 10),
+            'about': 'Update Test About',
+            'profile_pic': test_profile.profile_pic,
+            'resume': test_profile.resume}
+        response = client.post(update_profile_path, new_fields)
+        assert response.status_code == 302
```
Comment (Yarboa) [inline personal_profile/tests.py:126]: Please check the redirect uri of the response with another request(response.uri) 
Assert the 200 returned and templates

**Review by Yarboa — COMMENTED** [review]
Nice work. see my comment in test

**Review by Yarboa — APPROVED** [review]

**Pull request merged**